### PR TITLE
technical documentation pull request

### PR DESCRIPTION
**Pull request by belmeetmule**
added project files

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+#main-doc {
+  display: block;
+  margin: auto;
+  border-radius: 5px;
+  background: #333;
+}
+
+#navbar {
+  width: 25%;
```
Comment (belmeetmule) [inline css/style.css:28]: I guess it would be good to add css rules to make your side nav bar scroll-able. You may try overflow-y: scroll

**Review by belmeetmule — COMMENTED** [review]
Change required ♻️ 

It has been a massive work. Please find my comments for some parts of your code and make the required changes accordingly. 

Please submit for a re-review.
Thank you !

File: css/style.css (hunk 1 above)
Comment (Lucas-Masaba) [reply css/style.css:28]: - [ ] You are required to write at least to review comments and currently, this is the only one. Please add at least one more comment 👍

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @belmeetmule  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Use of inline comments 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- I encourage the use of emojis 🏄 🏂 . The use of emojis makes code reviews lighter to read by the recipient. One emoji could even express a whole sentence.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+@media (max-width: 460px) {
+  #content-wrapper {
+    font-size: 1rem;
+    line-height: 1;
+    float: clear;
+  }
+}
+
```
Comment (belmeetmule) [inline css/style.css:78]: I guess this is a duplicate of your css rules from line 51 - 57 of this same file. So, you should remove either of the two.

**Review by belmeetmule — COMMENTED** [review]
Changes are required !! ♻️ 

Great effort and you are almost there. Please work on the required changes and submit for a pull request. 

Thank you !

File: css/style.css (hunk 2 above)
Comment (1GORDON) [reply css/style.css:78]: Great job @belmeetmule 🤝

- [ ] [Optional] Well explained comments. A little politeness could do wonders. ✔️As stated earlier by the previous reviewer, you may please try to be more human-friendly by using emojis.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @belmeetmule 



Your project is complete!  There is nothing else to say other than... it's time to merge it : ship it:
Congratulations! :tada: :clap: :clap: :100:

## Top Highlights
- Good use of inline comments ✔️
- Great use of Github review actions ✔️
## Optional suggestions

- [ ] [Optional] Please try to use emojis so as to be more human-friendly.👏

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**